extensions/amp-bind/0.1/test/test-bind-integration-ptag.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createFixtureIframe} from '../../../../testing/iframe';
+import {bindForDoc} from '../../../../src/bind';
```

Comment:
I think if you change the import location to `'../../../../src/services`the continuous integration will pass and you can get this PR in :+1: